README.md

```diff
@@ -255,13 +257,13 @@ __Note__: reload is only used in development mode.
 
 ## Testing/Formatting/Linting
 ```
-isort -y
-black src tests
-pydocstyle src
-pytest --flake8 --isort
+poetry run black src tests
+poetry run pydocstyle src
+poetry run pytest
 ```
 
 To get code coverage information, you can run pytest directly.
+# WIP
```

Comment:
Is this heading meant to be here?

Reply:
This PR is still WIP, I will assign reviewers when it is ready (and I will remove all WIP sections before that)

Reply:
Now this PR is ready for review. Removed this `WIP` commend.

Reply:
This is still showing up the diff.

Reply:
Whoops, thought I had removed this. Now it's removed.